Cargo.toml

```diff
@@ -24,16 +24,12 @@ decimal128 = ["decimal"]
 name = "bson"
 
 [dependencies]
-byteorder = "1"
 chrono = "0.4"
-libc = "0.2"
 rand = "0.7"
 serde = "1.0"
 serde_json = { version = "1.0", features = ["preserve_order"] }
-time = "0.1"
 linked-hash-map = "0.5"
 hex = "0.3"
```

Comment:
Hey @AdminXVII would you be able to update `hex` to 0.4, in case your PR gets merged? I wanted to update it, but then saw your PR

Reply:
Yeah I think I could update the PR during the week. I'll get back to you soon.

Reply:
It was easier than I thought, it's already rebased. Tell me if there is anything you want me to do.